modules/acquia_cms_common/composer.json

```diff
@@ -22,7 +22,7 @@
         "drupal/moderation_sidebar": "^1.7",
         "drupal/password_policy": "^4.0",
         "drupal/pathauto": "1.11",
-        "drupal/redirect": "1.8",
+        "drupal/redirect": "^1.9",
```

Comment:
Is this the scope of the ticket?

Reply:
I don't think this change is related to this ticket.

Reply:
This is cherry-picked from ACMS-1947, to make CI runs